### DISCUSSION TITLE
SimCLR train NX-Tent loss dips at regular intervals (related to batch size)

### DISCUSSION THREAD

**Discussion by elizastarr** [Lightning-AI/pytorch-lightning#8417] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I'm training an encoder using SimCLR on my own dataset, and my NX-Tent training loss suddenly dips to the (about) same value for just one iteration about every 25 epochs when I use a batch size of 256 (global batch size of 512 and 771 iterations/epoch) and every 10 epochs when I use a batch size of 350 (global 600 and 564 iterations/epoch). Any ideas for why this is happening would be very much appreciated!

Other settings:
- Adam optimizer with learning rate of 1e10
- Also happens with the LARS optimizer
- 10 epochs of linear warmup followed by cosine annealing for 90 epochs without restarts
- Also happens without a learning rate scheduler
- using SSL online evaluator
- Defined my own strong augmentations rather than using SimCLR's
- Shuffling the training data

<img width="293" alt="image001" src="https://user-images.githubusercontent.com/8010740/125618665-f4a66f17-057b-46e2-be1c-11eaf4aaea2f.png">

Thanks!!

**Comment by tchaton** [comment]
@ananyahjha93 Any idea ?
